website/docs/contributing/release.md

```diff
@@ -65,6 +65,8 @@ OpenDAL will release all packages in the same version.
 - Bump version in `Cargo.toml`.
 - For the nodejs binding, bump version in `bindings/nodejs/package.json` and `bindings/nodejs/npm/*/package.json`.
 - For the java binding, bump version in `bindings/java/pom.xml`.
+- For the cpp binding, bump version in `bindings/cpp/CMakeLists.txt`.
+- For the haskell binding, bump version and update `source-repository this` field in `bindings/haskell/opendal.cabal`.
 - Update `CHANGELOG.md`, refer to [Generate Release Note](reference/generate_release_note.md) for more information.
 - If there are breaking changes in `core`, please also update `core/src/docs/upgrade.md`.
 - After all those changes, submit a PR with name `Bump to version x.y.z`.
```

Comment:
The bump version is become longer over days. How about maintain an issue template here? 

Like what we does for VOTE mails, we can maintain a template like, for example:


---

Create a new issue with the following content.

## Title

```
Tracking issuses of OpenDAL ${opendal_version} Release
```

## Content

```
This issue is used to track blockers of the opnedal ${opendal_version} release.

## Tasks

- [ ] Bump versions across projects
  - [ ] Rust Core: Bump version in `Cargo.toml`
  - [ ] Nodejs Binding: xxx
  - [ ] Python Binding: xxx
  - [ ] ...
- [ ] Package release candidate.
- [ ] Start VOTE at opendal community
- [ ] Start VOTE at incubator community
- [ ] Send annouance
```